docs/conf.py

```diff
@@ -14,7 +14,7 @@
 # FOR! DO NOT LET THEM BE TARGETTED AT MORE THAN ONE MINOR SERIES!
 # I.E.: 0.1.x -- important to add 'dev' suffix for docs targetting development
 # series.
-DISPLAY_VERSION = "0.13.0"
+DISPLAY_VERSION = "0.13.1"
```

Comment:
I believe the outcome  of discussion in https://github.com/learningequality/kolibri-docs/issues/48 was that we would make this `0.13` and leave off the patch release?

Reply:
My bad, I wasn't following that discussion :woman_facepalming: